### PR TITLE
feat: add diagnosis-key-entity-table-indexes

### PR DESCRIPTION
**Pull request by dfischer-tech**

### CODE REVIEW

File: src/main/resources/db/changelog/v001-add-diagnosis-key-entity-indexes.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+              - column:
+                  name: uploader_information_batch_tag
+                  type: varchar(64)
+        - createIndex:
+            tableName: diagnosiskey
+            indexName: idx_diagnosiskey_batch_tag
+            columns:
+              - column:
+                  name: batch_tag
+                  type: varchar(64)
```
Comment (f11h) [inline src/main/resources/db/changelog/v001-add-diagnosis-key-entity-indexes.yml:19]:
```suggestion
```

File: src/main/resources/db/changelog/v001-add-diagnosis-key-entity-indexes.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      id: create-diagnosis-key-entity-table-indexes
+      author: dfischer-tech
+      changes:
+        - createIndex:
+            tableName: diagnosiskey
+            indexName: idx_diagnosiskey_uploader_information_batch_tag
+            columns:
+              - column:
+                  name: uploader_information_batch_tag
+                  type: varchar(64)
```
Comment (f11h) [inline src/main/resources/db/changelog/v001-add-diagnosis-key-entity-indexes.yml:12]: As per [liquibase documentation](https://docs.liquibase.com/change-types/community/create-index.html)  the type seems to be redundant.

```suggestion
```

**Review by f11h — CHANGES_REQUESTED** [review]
Except the little thing regarding type property it looks good to me 👍

**Review by mlaue-tech — APPROVED** [review]
Thanks for your contribution!  🐣

**Review by f11h — APPROVED** [review]

**Pull request merged**